### DISCUSSION TITLE
Rpi 5 - kleine grüne led

### DISCUSSION THREAD

**Discussion by ghost** [OpenCCU/OpenCCU#2689] [🇩🇪 Fragen&Antworten] [answered]
Raspberry 5 hat ein Problem mit dem Ein- und Ausschalten der kleinen grünen LED im version 3.75.6.20240319

das funktioniert gut: `system.Exec("echo heartbeat >/sys/class/leds/ACT/trigger"); `
das funktioniert nicht : `system.Exec("echo none >/sys/class/leds/ACT/trigger");`

Anstatt es einzuschalten, schaltet es es aus: `system.Exec("echo 1 >/sys/class/leds/ACT/brightness");`

und anstatt es auszuschalten, schaltet es es ein: `system.Exec("echo 0 >/sys/class/leds/ACT/brightness");`

**Comment by jens-maus** [comment, marked as answer]
Noch nicht selber probiert, aber wenn das so ist ist das ein "upstream" Problem im Linux kernel des RaspberryPi und da können wir da nix machen ausser das dort zu melden.

Würde vorschlagen du bootest mal ein Standard RaspberryPiOS darauf und schaust wie sich das verhält.

**Reply by ghost** [reply]
Das habe ich auch gedacht, dass es sich um ein Problem des Linux-Kernels handelt. 
Gestern habe ich eine neue Station für den RaspberryMatic gebaut und alles funktioniert tip top
  
https://homematic-forum.de/forum/viewtopic.php?f=65&t=80283&start=100#p799768

und es tut mir leid, aber ich habe nicht die Kraft, es auseinander zu nehmen und auf einem anderen System zu testen. Ich bin sehr zufrieden mit dem, was ich jetzt habe. Danke Jens !
